doc/src/manual/parallel-computing.md

```diff
@@ -258,11 +258,15 @@ snippet:
 
 ```julia-repl
 A = rand(10,10)
-remotecall_fetch(()->foo(A), 2)
+remotecall_fetch(()->norm(A), 2)
 ```
 
+In this case [`norm`](@ref) is a function that takes 2D array as a parameter, and MUST be defined in
```

Comment:
As someone who found the docs confusing, I think it is necessary.  Someone might think that `norm` is the only function one can use to push a global to a worker.